Porto-Buttons/schema.json

```diff
@@ -3,13 +3,11 @@
   "properties": {
     "Button": {
       "type": "string",
-      "title": "Button Text (optional)",
-      "required": false
+      "title": "Button Text (required)"
     },
     "Icon": {
       "type": "string",
-      "title": "Icon (optional) - Example: fas fa-android",
-      "helper": "Find values at: <a href=\"https://fontawesome.com/icons?d=gallery\" target=\"_blank\">FontAwesome</a>"
```

Comment:
@alejoroman0605 Shouldn't we still have a helper here?  (and for any/all fields)

Reply:
@WillStrohl  yes we must have it, I didn't realize